index.js

```diff
@@ -1,109 +1,145 @@
 #!/usr/bin/env node
 
-import fs from 'fs';
-import path from 'path';
+import 'babel-polyfill';
```

Comment:
I know you've already switched to node 8, which supports async/await out of box. However, if we keep this polyfill, we'll be able to support node 6 as well as it's still in maintenance till April 2019.